pkg/ibmcsidriver/ibm_csi_driver.go

```diff
@@ -105,6 +108,13 @@ func (icDriver *IBMCSIDriver) SetupIBMCSIDriver(provider cloudProvider.CloudProv
 
 	icDriver.logger.Info("Successfully setup IBM CSI driver")
 
+	// Set up Region
+	regionMetadata, err := nodeMeta(os.Getenv("KUBE_NODE_NAME"), lgr)
```

Comment:
can this nodeMeta be nil , please handle for nil condition

Reply:
https://github.com/IBM/ibm-csi-common/blob/master/pkg/metadata/metadata.go#L54
From this method, even if it is nil, error is returned. so we are handling the error before proceeding